core/src/com/unciv/models/ruleset/unit/UnitType.kt

```diff
@@ -71,4 +75,12 @@ enum class UnitType{
     fun isMissileUnit():Boolean{
         return this == Missile
     }
+
+    fun isAircraftCarrierUnit():Boolean{
+        return this == WaterAircraftCarrier
+    }
+
+    fun isMissileCarrierUnit():Boolean{
+        return this == WaterMissileCarrier
```

Comment:
As far as I remember the submarines are also `Missile carrier units`, so they should be added here.

Reply:
You're correct, but I think you were thinking about [Nuclear Submarines](https://civilization.fandom.com/wiki/Nuclear_Submarine_(Civ5)). The predecessor [Submarines](https://civilization.fandom.com/wiki/Submarine_(Civ5)) don't carry missiles.

Also, just noting down so I remember since I was on the wiki, [Missile Cruisers](https://civilization.fandom.com/wiki/Missile_Cruiser_(Civ5)) are another type of missile carrier.